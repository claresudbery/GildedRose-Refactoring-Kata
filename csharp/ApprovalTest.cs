using System;
using System.IO;
using System.Text;
using ApprovalTests;
using ApprovalTests.Combinations;
using ApprovalTests.Reporters;
using NUnit.Framework;

namespace csharp
{
    [UseReporter(typeof(DiffReporter))]
    [TestFixture]
    public class ApprovalTest
    {
    }
}
