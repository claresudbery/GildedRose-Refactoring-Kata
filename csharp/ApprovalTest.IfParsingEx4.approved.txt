[green,green] => True
[green,blue] => True
[blue,green] => True
[blue,blue] => True
